documentation/en/source/pages/command-line/scan.rst

```diff
@@ -35,6 +35,14 @@ Deletes an existing scan. The options are:
 	* ``--scantype`` (mandatory): the type to be deleted. Can be one of: instance, scan, or all. When you set the type to ``instance``, the instance and all scans linked to it will be deleted unless using the ``scansonly`` flag. When you specify the type as ``scan`` only the scan with the ID to specify will be deleted. If you set the type to ``all``, all the instances and scans on your UForge will be deleted (regardeless of the ``id`` you set).
```

Comment:
formatting review
Can be one of: (use two ` before and after each option) ``instance``, ``scan``, or ``all``. 
When you specify the type as ``scan`` only the scan with the ID specified will be deleted.